src/main/java/org/bonitasoft/engine/connector/uipath/UIPathService.java

```diff
@@ -34,17 +30,20 @@
 import retrofit2.http.POST;
 import retrofit2.http.Path;
 
+import java.util.List;
+import java.util.Map;
+
 public interface UIPathService {
 
     @FormUrlEncoded
     @POST("api/account/authenticate")
     Call<Map<String, String>> authenticate(@Field("tenancyName") String tenant,
-            @Field("usernameOrEmailAddress") String user,
-            @Field("password") String password);
+                                           @Field("usernameOrEmailAddress") String user,
```

Comment:
Looks a bit over indented no?

Reply:
this part is not modified by this PR, except formating